umm-lib/src/cmr/umm/dif/collection/collection_association.clj

```diff
@@ -1,8 +1,9 @@
 (ns cmr.umm.dif.collection.collection-association
   "Provide functions to parse and generate DIF collection associations."
-  (:require [clojure.data.xml :as x]
-            [cmr.common.xml :as cx]
-            [cmr.umm.umm-collection :as c]))
+  (:require
+   [clojure.data.xml :as x]
```

Comment:
if we are in here and doing annoying things, I suggest changing x to xml and c to umm-c or something longer then one letter.

Reply:
well, take this for what it is worth, there are a lot of these

Reply:
Yes, there are a lot of these, and I don't really want to since this code will be deleted in the future, i will update all of the single aliases for all of the files that I am changing.

Reply:
Obligatory: clj-kondo has a default config for consistent and standardized alias names for common requires ("string", "xml", etc) that we could pull in.

Reply:
I still do not like them, but in this case, I have been removing them in short files and leaving them in large ones.